README.md

```diff
@@ -1,104 +1,181 @@
+- [Refunds](https://www.mollie.com/en/docs/reference/refunds/list-all): allow you to make refunds in relation to a payment.
+- [Customers](https://www.mollie.com/en/docs/reference/customers/create): allow you to manage your customer's details.
+- [Mandates](https://www.mollie.com/en/docs/reference/mandates/create): allow you to charge a customer's credit card or bank account recurrently.
+- [Subscriptions](https://www.mollie.com/en/docs/reference/subscriptions/create): allow you to schedule recurring payments to take place at regular intervals.
 
-Accepting [iDEAL](https://www.mollie.com/en/payments/ideal/), [Bancontact/Mister Cash](https://www.mollie.com/en/payments/bancontact/), [SOFORT Banking](https://www.mollie.com/en/payments/sofort/), [Creditcard](https://www.mollie.com/en/payments/credit-card/), [SEPA Bank transfer](https://www.mollie.com/en/payments/bank-transfer/), [SEPA Direct debit](https://www.mollie.com/en/payments/direct-debit/), [Bitcoin](https://www.mollie.com/en/payments/bitcoin/), [PayPal](https://www.mollie.com/en/payments/paypal/), [Belfius Direct Net](https://www.mollie.com/en/payments/belfius/), [KBC/CBC](https://www.mollie.com/en/payments/kbc-cbc/), [paysafecard](https://www.mollie.com/en/payments/paysafecard/), [ING Home'Pay](https://www.mollie.com/en/payments/ing-homepay/), [Giftcards](https://www.mollie.com/en/payments/gift-cards/), [Giropay](https://www.mollie.com/en/payments/giropay/) and [EPS](https://www.mollie.com/en/payments/eps/) online payments without fixed monthly costs or any punishing registration procedures. Just use the Mollie API to receive payments directly on your website or easily refund transactions to your customers.
+# Getting started
+
+- [Requirements](#requirements)
+- [Installation](#installation)
+- [Importing the client](#importing-the-client)
+- [Authentication](#authentication)
+- [Making your first request](#making-your-first-request)
+- [Documentation](#documentation)
+
+## Requirements
 
-## Requirements ##
 To use the Mollie API client, the following things are required:
 
-+ Get yourself a free [Mollie account](https://www.mollie.nl/aanmelden). No sign up costs.
-+ Create a new [Website profile](https://www.mollie.nl/beheer/account/profielen/) to generate API keys (live and test mode) and setup your webhook.
++ Get yourself a free [Mollie account](https://www.mollie.com/dashboard/signup). No sign up costs.
++ Login to your [Dashboard](https://www.mollie.com/dashboard) to get your API keys (live and test mode).
 + Now you're ready to use the Mollie API client in test mode.
-+ In order to accept payments in live mode, payment methods must be activated in your account. Follow [a few of steps](https://www.mollie.nl/beheer/diensten), and let us handle the rest.
++ In order to accept payments in live mode, payment methods must be activated in your account. Just follow [a few steps](https://www.mollie.com/dashboard/?modal=onboarding) and let us handle the rest.
```

Comment:
I think the onboarding endpoint has changed to https://www.mollie.com/dashboard/onboarding